frontend-shared/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@hypothesis/frontend-shared",
+  "version": "1.0.0",
+  "repository": "hypothesis/client/frontend-shared",
+  "dependencies": {
+    "cross-env": "5.0.5"
```

Comment:
this is required for yarn workspace

Reply:
What happens if you don't list `cross-env` here?